### DISCUSSION TITLE
Missing io.featurehub.tiles

### DISCUSSION THREAD

**Discussion by LucioRebora** [featurehub-io/featurehub#338] [Q&A] [answered]
Hello, i have problems when excecute `mvn clean install` for run in local.
I use Java 11
Can you help me please?

`featurehub git:(master) ✗ mvn clean install
[INFO] Scanning for projects...
[INFO] gitflow-incremental-builder is disabled.
[ERROR] Missing:
[ERROR] ----------
[ERROR] 1) io.featurehub.tiles:tile-java:xml:[1.1,2)
[ERROR]
[ERROR]   Try downloading the file manually from the project website.
[ERROR]
[ERROR]   Then, install it using the command:
[ERROR]       mvn install:install-file -DgroupId=io.featurehub.tiles -DartifactId=tile-java -Dversion=[1.1,2) -Dpackaging=xml -Dfile=/path/to/file
[ERROR]
[ERROR]   Alternatively, if you host your own repository you can deploy the file there:
[ERROR]       mvn deploy:deploy-file -DgroupId=io.featurehub.tiles -DartifactId=tile-java -Dversion=[1.1,2) -Dpackaging=xml -Dfile=/path/to/file -Durl=[url] -DrepositoryId=[id]
[ERROR]
[ERROR] ----------
[ERROR] 1 required artifact is missing.
[ERROR]
[ERROR] for artifact:
[ERROR]   io.featurehub.tiles:tile-java:xml:[1.1,2)
[ERROR]
[ERROR] from the specified remote repositories:
[ERROR]   central (https://repo.maven.apache.org/maven2, releases=true, snapshots=false)
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException`

**Comment by rvowles** [comment]
Hey, just checking you saw this https://docs.featurehub.io/developers.html

**Reply by LucioRebora** [reply]
yes, but it doesn't work for me. thank you anyway

**Comment by vinothank95** [comment]
I had same error. i have copied plugins details from  tile-java.xml and put it in pom.xml manually. its worked

**Reply by LucioRebora** [reply]
I don't have tile-java:xml. thanks for your time

**Comment by rvowles** [comment]
hi - can you try this?

from your checkout, type:

cd backend

mvn -f pom-first.xml clean install

then go back up one level and

mvn -T4C clean install

**Reply by LucioRebora** [reply]
Hi, something we advance. Crash creating "edge 1.1-SNAPSHOT". I start to investigate. Thank you very much for the help.
![image](https://user-images.githubusercontent.com/26748830/116500366-d2293f00-a884-11eb-9d51-0a8563ce345d.png)

**Comment by rvowles** [comment, marked as answer]
Can you go into the backend/sse-edge project and try a d mvn clean install
and if it fails please put the output in this discussion? Thanks!

On Thu, Apr 29, 2021 at 3:54 PM LucioRebora ***@***.***>
wrote:

> Hi, something we advance. Crash creating "edge 1.1-SNAPSHOT". I start to
> investigate. Thank you very much for the help.
> [image: image]
> <https://user-images.githubusercontent.com/26748830/116500366-d2293f00-a884-11eb-9d51-0a8563ce345d.png>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/featurehub-io/featurehub/discussions/338#discussioncomment-672972>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAANTWFYHK6VK2RXGYJBJQTTLDJ6ZANCNFSM43WHH4MA>
> .
>


-- 
---
Richard Vowles,
Full stack - from Kubernetes, through Node & Java, Web and Mobile
development in Flutter - software developer for hire!

ph: +64275467747

**Reply by LucioRebora** [reply]
![image](https://user-images.githubusercontent.com/26748830/116549818-58b53f00-a8cc-11eb-9ab7-35fb048f021a.png)

**Reply by LucioRebora** [reply]
Fix it by removing -SNAPSHOT.  Thank you very much for your help

![image](https://user-images.githubusercontent.com/26748830/116632959-b4181900-a92e-11eb-97d2-fe7958a73a12.png)

**Reply by rvowles** [reply]
Thanks, it's not right that it can't resolve that, It must not be listed in the reactor...

**Reply by IrinaSouth** [reply]
@rvowles is this now resolved? any changes needed in the docs?

**Reply by rvowles** [reply]
Not that i'm aware of, I'm still not sure why it would have not build once the tiles were installed.

**Reply by IrinaSouth** [reply]
has been fixed now, tested and merged
